etl/src/main/java/org/pih/hivmigration/etl/sql/VitalsMigrator.groovy

```diff
@@ -0,0 +1,85 @@
+package org.pih.hivmigration.etl.sql
+
+class VitalsMigrator extends SqlMigrator {
+    @Override
+    def void migrate() {
+
+        executeMysql("Create vitals staging table", '''
+            CREATE TABLE hivmigration_vitals (
+              source_encounter_id int,
+              sign varchar(30),
+              result double,
+              result_unit varchar(100),
+              source_patient_id int,
+              KEY `source_encounter_idx` (`source_encounter_id`)
+            );
+        ''')
+
+        // TODO: should we do something with the 563 vitals with encounter type 'food_study'
+        //      or the 3 with type patient_contact?
+        loadFromOracleToMySql('''
+            INSERT INTO hivmigration_vitals (source_encounter_id, sign, result, result_unit, source_patient_id)
+            VALUES (?, ?, ?, ?, ?)
+        ''', '''
+            select 
+                v.ENCOUNTER_ID, 
+                v.SIGN, 
+                v.RESULT, 
+                v.RESULT_UNIT,
+                e.PATIENT_ID
+            from HIV_EXAM_VITAL_SIGNS v, HIV_ENCOUNTERS e
+            where v.ENCOUNTER_ID = e.ENCOUNTER_ID 
+            and SIGN in ('height', 'weight', 'bmi')
+            and e.TYPE in ('intake', 'followup')
+            and e.PATIENT_ID is not null
+        ''')
+
```

Comment:
We'll want to first create new Vitals encounters here, or we should add explicitly into the EncounterMigrator to ensure it is already in place.

Reply:
So, yes, assuming that we are sticking with our decision to store these in Vitals encounters, I'm thinking here that we have to create a new Vitals encounter for each source_encounter_id in the hivmigration_vitals table?  

Seems like we'd have to do this here, not in the EncounterMigrator.  If we did do this, we'd want the VisitMigrator to run after this (which I don't think should be an issue).